frame/staking/README.md

```diff
@@ -92,11 +92,11 @@ An account can become a nominator via the [`nominate`](https://docs.rs/pallet-st
 The **reward and slashing** procedure is the core of the Staking module, attempting to _embrace
 valid behavior_ while _punishing any misbehavior or lack of availability_.
 
-Rewards must be claimed for each era before it gets too old by `$HISTORY_DEPTH` using the
+Rewards must be claimed for each era before it gets too old by [`HistoryDepth`] using the
 `payout_stakers` call. Any account can call `payout_stakers`, which pays the reward to the
```

Comment:
```suggestion
Rewards must be claimed on a per-era basis via the `payout_stakers` call, and must be claimed before [`HistoryDepth`] eras have passed since the reward's inclusion, after which any unclaimed rewards will expire and be removed from storage. Any account can call `payout_stakers`, which pays the reward to the
```